src/librustdoc/clean/types.rs

```diff
@@ -1609,33 +1610,37 @@ crate enum VariantKind {
     Struct(VariantStruct),
 }
 
+/// Small wrapper around `rustc_span::Span` that adds helper methods.
 #[derive(Clone, Debug)]
 crate struct Span {
```

Comment:
```suggestion
crate struct Span(rustc_span::Span);
```

What do you think? A struct with one field could just be written as a wrapper type. At first I was thinking that we could simply drop it but I think that in the end, it's better to keep the distinction because it makes it easier to understand what's being used.

Reply:
Sure, writing this as a tuple seems fine. I like the wrapper because it enforces calling `source_callsite()`, you can't forget.